docs/reference/shutdown/apis/shutdown-delete.asciidoc

```diff
@@ -26,7 +26,8 @@ you must be an operator to use this API.
 Enables a node to resume normal operations following a
 <<put-shutdown, put shutdown>> request.
 You must explicitly clear the shutdown request when a node rejoins the cluster,
-regardless of the shutdown type.
+regardless of the shutdown type. When node is permanently removed (cluster shrink)
+the node shutdown request also has to be cleared.
```

Comment:
I agree this needs to be mentioned explicitly. What do you think of this wording?

```suggestion
or when a node has permanently left the cluster. Shutdown requests are never
removed automatically by {es}.
```